docs-page.html

```diff
@@ -144,10 +145,178 @@ <h2>Example of Avatar</h2>
+                            sandbox="allow-scripts allow-same-origin">
+                        </iframe>
+                    </div>
+                </div>
+                <!-- *** -->
+            </div>
+            <hr />
+            <!-- end of Badge -->
+
+            <!-- Buttons -->
+            <div id="compo-buttons">
+                <h1>Buttons</h1>
+                <h3>
+                    Buttons are also called as call to action. We have range of buttons and their states. You may use a tag or button element, you need to add respective classes, and you are good to go.
+                </h3>
+                <!-- primary solid button -->
+                <h2>Example of Solid Button Styles</h2>
+                <p>
+                    Whenever you want your user to have CTA (call to action) on a link or button, use primary style buttons.
+                </p>
+                <p>As you want to use success button, then use the class accordingly. (For e.g <span class="highlight-text">class="btn solid-success"</span></p>
+                <p>Check code below to copy the html part as follow below.</p>
+                <div class="solid-buttons-example">
+                    <a class="btn solid-primary">Primary</a>
```

Comment:
links are designed as buttons here i guess, buttons are for some action(with routing sometimes) and links are only for routing